slices/xfonts-encodings.yaml

```diff
@@ -0,0 +1,64 @@
+package: xfonts-encodings
+
+essential:
+  - xfonts-utils_copyright
+
+slices:
+  encodings:
+    contents:
```

Comment:
Do we not need `x11-common`?

Reply:
It's of course fine if we do not. Just asking for info.

Reply:
It is not a hard dependency (no binaries we really need there).